Tests/TuistCacheTests/ContentHashing/CoreDataModelsContentHasherTests.swift

```diff
@@ -0,0 +1,91 @@
+import Basic
+import Foundation
+import TuistCore
+import TuistCoreTesting
+import XCTest
+import TuistSupport
+import TuistCacheTesting
+@testable import TuistCache
+@testable import TuistSupportTesting
+
+
+final class CoreDataModelsContentHasherTests: TuistUnitTestCase {
+    private var sut: CoreDataModelsContentHasher!
+    private var coreDataModel: CoreDataModel!
+    private var temporaryDirectory: TemporaryDirectory!
+    private var mockContentHasher: MockContentHashing!
+    private let defaultValuesHash = "05c9d517e2cf12b45786787dae929a23" // Expected hash for the CoreDataModel created with the buildCoreDataModel function using default values
+    override func setUp() {
```

Comment:
You can use [this one](https://developer.apple.com/documentation/xctest/xctest/3521150-setupwitherror) instead and get rid of the `do {} catch {}`

Reply:
wow, I didn't know this one, nice!

Reply:
It's only Swift 5.2 Xcode 11.4 though

Reply:
Ignore what I said then.